src/Model/FormProcessing.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Model;
+
+class FormProcessing
+{
+
+
+    /**
+     * ! VERIFY IF $_POST IS EMPTY
+     */
+    public function verifyEmptyPost(): array
+    {
+        $errors = [];
+        foreach ($_POST as $value) {
+            if (empty($value)) {
+                $errors[] = 'Merci de remplir tous les champs';
+            }
+        }
+        return $errors;
+    }
+
+    public function verifyGetToFilter(): array
+    {
+
+        $errors = [];
+        if (!empty($_GET)) {
+            $items = [
+                'author_id' => $_GET['author_id'],
+                'editor_id' => $_GET['editor_id'],
+                'category_id' => $_GET['category_id'],
+                'format_id' => $_GET['format_id'],
+                'location_id' => $_GET['location_id'],
+                'status_id' => $_GET['status_id'],
+            ];
```

Comment:
You should verify the presence (and emptiness eventually)  of each $_GET field individually